src/pages/movieDetailing.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+import {renderPageLayout} from '../layouts/createLayout.js';
+
+export function createMovieInfo() {
```

Comment:
@Mmarvinn Task wasn't about MovieInfo, why did you put it in this PR?

Reply:
sorry, my bad, in next time i didn`t will be put other files in PR